lib/mongoid/findable.rb

```diff
@@ -60,6 +60,18 @@ def count
       with_default_scope.count
     end
 
+    # Returns an estimated count of records in the database.
+    # If you want to specify conditions use where.
+    #
+    # @example Get the count of matching documents.
+    #   Person.estimated_count
+    #   Person.where(title: "Sir").estimated_count
+    #
+    # @return [ Integer ] The number of matching documents.
+    def estimated_count
+      with_default_scope.estimated_count
```

Comment:
I believe calling estimated_count on a class with a default scope defined needs to be tested as I don't expect it to work (at all or correctly).

Reply:
There is a single test for estimated count that does pass... At least we know it doesn't crash... I'm not sure how to test it because I'm not sure what the outcome of the function is supposed to be (after all it is estimated)...

Reply:
Which test uses a default scope?

Reply:
See my comment below

Reply:
I believe if a default scope is defined, estimated_count should raise an error, unless you would like to make an argument to the contrary.